neuland/util/neuland_add_random_hitpar.py

```diff
@@ -200,9 +212,7 @@ def __assign_random_values(self, module_id: int):
         energy_gain = self.generate_random_value_with_cos(self.energy_gain, module_id)
         pedestal = self.generate_random_value_with_cos(self.pedestal, module_id)
         pmt_thresh = self.generate_random_value_with_cos(self.pmt_thresh, module_id)
-        saturation_coefficient = self.generate_random_value_with_cos(
-            self.saturation_coefficient, module_id
-        )
+        saturation_coefficient = energy_gain * (1.75*10**(-3))
```

Comment:
Don't change this. saturation_coefficient and energy_gain should be set independently.

If they should be correlated, this should be done outside of this class.